src/renderer/services/deposit/DepositTransactionService.ts

```diff
@@ -0,0 +1,97 @@
+import {Keypair as KeyPair} from "@chainsafe/bls/lib/keypair";
+import bls from "@chainsafe/bls";
+import {BLSPubkey as BLSPubKey, bytes, DepositData} from "@chainsafe/eth2.0-types";
+import {createHash} from "crypto";
+import BN from "bn.js";
+import {signingRoot} from "@chainsafe/ssz";
+import {BLSDomain} from "@chainsafe/bls/lib/types";
+import abi from "ethereumjs-abi";
+import {config} from "@chainsafe/eth2.0-config/lib/presets/mainnet";
+import {IDepositParams, ITx} from "./types";
+import {Wallet} from "ethers";
+import {TransactionRequest} from "ethers/providers";
+
+// fixed deposit amount
+const depositAmountInEth = 32;
+
+// fixed BLS domain -> deposit = 3
+const depositBLSDomain: BLSDomain =
+    // domain_type + fork_version
+    Buffer.concat([new BN(3).toArrayLike(Buffer, "le", 4), Buffer.alloc(4)]);
+
+// definition of contracts deposit function
+const depositFunctionSignature = "deposit(bytes,bytes,bytes,bytes32)";
```

Comment:
@mpetrunic maybe move these constants to `options.ts` file.

Reply:
yeah move those to constant.ts file.

Reply:
you could generate this from abi